.github/workflows/push.yaml

```diff
@@ -69,7 +73,7 @@ jobs:
           # list of Docker images to use as base name for tags
           images: ${{secrets.IMAGE_NAME}}
           flavor: |
-            suffix=${{matrix.flavor}}
+            suffix=${{matrix.php-version}}-${{matrix.flavor}}
```

Comment:
You should also add this to the test workflow as well so we can see what the output is when it builds this PR. I'm not 100% sure, but I believe this change would prevent `latest` from being updated.